plugins/API/tests/System/expected/test_AutoSuggestAPITest__Live.getLastVisitsDetails_range.xml

```diff
@@ -2125,10 +2125,10 @@
 	</row>
 	<row>
 		<idSite>1</idSite>
-		<idVisit>8</idVisit>
-		<visitIp>1.2.4.3</visitIp>
+		<idVisit>30</idVisit>
```

Comment:
_Picked a semi random file just to have this as a comment thread._

The change in [`plugin/CustomVariables`](https://github.com/matomo-org/plugin-CustomVariables/compare/fixturechange) should be merged in the plugin with a following submodule update.